### ISSUE TITLE
Cayenne Pepper And Water Diet - Fad or Fiction?

### ISSUE THREAD

**Issue by nadjibandnaaima** [nadjibandnaaima/diets-for-quick-weight-loss#627]
<div dir="ltr" style="text-align: left;">
<b><span style="font-size: x-large;">Cayenne Pepper And Water Diet - Fad or Fiction?</span></b><br><br>
Have you heard of the cayenne pepper and water diet? Controversy has recently surrounded the so-called cayenne pepper and water diet, primarily due to it's association with Beyonce Knowles who apparently lost 20 pounds during filming for the movie "Dreamgirls" living only on cayenne pepper and water. The facts are not in dispute, the actress did apparently lose weight quickly over a short period and there is no reason to dispute how this was achieved.<br><br>
However the question that must be asked is can this be called a "diet"? Some would say that what Beyonce Knowles did was simply starve herself of food, surviving solely on liquids. Surely the cayenne pepper had no noticeable effects other than perhaps adding a little taste to the water?<br><br>
It should perhaps be noted that there are certainly recorded health benefits associated with cayenne pepper, including aiding digestion, strengthening of the heart, claims it contains mild pain relieving qualities, and there are even reports of using cayenne as a dressing for wounds.<br><br>
Even taking these reported benefits into account, can this cayenne pepper and water diet really be called a diet? It should more properly be seen as a purely starvation exercise undertaken by a committed actor wishing to lose weight quickly in pursuit of her art and career, presumably with the safety net afforded to highly paid and closely monitored movie stars. Presumably she will have been surrounded by assistants and helpers who would have been immediately on hand had Beyonce suffered from any starvation effects or ill health.<br><br>
The danger in this story as with other fad diets is that impressionable people, young girls in particular, will have heard about this seemingly miraculous cayenne pepper and water diet and will wish to try it themselves, without any regard to the obvious and significant health dangers inherent in starving yourself in such a way. In particular this form of dieting could be very damaging if sustained for any length of time, any form of fasting should be carefully monitored for signs of deteriorating health.<br><br>
It should also be said that this story and the controversy it has created are not the fault of Beyonce Knowles, rather it is created by media hype attempting to profit from the possibility others may try and follow her example. In recent times other actors have been required to lose weight quickly for their future roles, for example Christian Bale in the movie The Machinist. The actor dropped his weight to a dangerously low level by reportedly eating a single apple and a tin of tuna every day, but this did not become the "apple and tuna" fad diet, probably because Christian Bale does not have the same widespread fashionable image as Beyonce.<br><br>
So in conclusion is this really a Diet? I would suggest not, with the hype surrounding it currently it could be categorized as a fad, and potentially a dangerous one. The only safe way to permanently lose weight is to eat healthy food, eat in moderation, and crucially undertake regular exercise. If you do this you will burn excess fat, your arteries will stay clear and healthy and you won't risk damaging your body by starving it of the food it needs to stay fit and healthy.<br><br><br>
</div>
<br><br>
via diets for quick weight loss http://ift.tt/1RPkSGN<br>
December 29, 2015 at 12:31PM
